tensorflow/python/debug/cli/curses_ui.py

```diff
@@ -855,7 +857,30 @@ def _on_textbox_keypress(self, x):
       except curses.error:
         mouse_event_type = None
 
-      if mouse_event_type == curses.BUTTON1_RELEASED:
+      if mouse_event_type == curses.BUTTON1_PRESSED:
+        # Logic for held mouse-triggered scrolling.
+        if mouse_x >= self._max_x - 2:
+          # Disable blocking on checking for user input.
+          self._command_window.nodelay(True)
+
+          # Loop while mouse button is pressed.
+          while mouse_event_type == curses.BUTTON1_PRESSED:
+            # Sleep for a bit.
+            curses.napms(self._MOUSE_SCROLL_DELAY_MS)
+            scroll_command = self._scroll_bar.get_click_command(mouse_y)
+            if scroll_command is not None:
```

Comment:
Can you change this to
```python
if scroll_command in (_SCROLL_UP_A_LINE, SCROLL_DOWN_A_LINE):
```

The behavior for hold-scrolling outside the UP and DN blocks is a little weird due to the way the multi-line scrolling actions happen.